node/bft/ledger-service/src/ledger.rs

```diff
@@ -182,16 +182,13 @@ impl<N: Network, C: ConsensusStorage<N>> LedgerService<N> for CoreLedgerService<
             (TransmissionID::Transaction(expected_transaction_id), Transmission::Transaction(transaction_data)) => {
                 match transaction_data.clone().deserialize_blocking() {
```

Comment:
it's not ideal that we do blocking deserialization in an `async` context; we should either do it before calling `check_transaction_basic` (with a deserialized transaction), or use a blocking task here